privaterelay/utils.py

```diff
@@ -40,17 +40,35 @@ def get_countries_info_from_request_and_mapping(
     }
 
 
+def get_countries_info_from_lang_and_mapping(
```

Comment:
Nit: this seems like overkill to determine if the user has the bundle plan available, but otherwise is OK.